backend/CHANGELOG.md

```
@@ -1,4 +1,16 @@
 # Changelog
+## [4.0.1] - 2025-04-22
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)